pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -255,6 +209,10 @@ spec:
           - name: pathInRepo
             value: tasks/push-snapshot/push-snapshot.yaml
       params:
+        - name: tagPrefix
+          value: "params.tagPrefix"
```

Comment:
This isn't the proper syntax even if we were passing more pipeline params

Reply:
:( fixed, but I understand we need to change this with your above comment.